src/overlays/actors/ovl_En_Dg/z_en_dg.c

```diff
@@ -909,69 +1061,86 @@ void func_8098B28C(EnDg* this, PlayState* play) {
         Actor_MoveWithGravity(&this->actor);
     }
 
-    func_80989974(this);
+    EnDg_PlaySfxWalk(this);
     Actor_PlaySfxAtPos(&this->actor, NA_SE_EV_SMALL_DOG_GROAN);
 }
 
-void func_8098B390(EnDg* this, PlayState* play) {
+/**
+ * An unused function that will make the dog bark at the player endlessly.
+ * It is quite similar to EnDg_BarkAtGoron; perhaps this is an earlier version of it?
+ * Unlike that function, this will make the dog approach the player if they move too far away.
+ * It also mixes in multiple growls into the bark.
+ */
+void EnDg_UnusedBarkAtPlayer(EnDg* this, PlayState* play) {
     if (this->actor.xzDistToPlayer < 150.0f) {
-        func_80989140(&this->skelAnime, sAnimations, 11);
-        this->actionFunc = func_8098B28C;
+        EnDg_ChangeAnimation(&this->skelAnime, sAnimations, DOG_ANIMATION_WALK_BACKWARDS);
+        this->actionFunc = EnDg_UnusedBackAwayFromPlayer;
     } else if (this->actor.xzDistToPlayer > 200.0f) {
-        func_80989140(&this->skelAnime, sAnimations, 1);
-        this->actionFunc = func_8098AE58;
+        EnDg_ChangeAnimation(&this->skelAnime, sAnimations, DOG_ANIMATION_WALK);
+        this->actionFunc = EnDg_UnusedWalkToPlayer;
     }
 
-    func_80989A9C(this, 0.0f);
-    func_80989A9C(this, 1.0f);
-    func_80989A9C(this, 2.0f);
-    func_80989A9C(this, 3.0f);
-    func_809899C8(this, 5.0f);
+    EnDg_PlaySfxGrowl(this, 0.0f);
+    EnDg_PlaySfxGrowl(this, 1.0f);
+    EnDg_PlaySfxGrowl(this, 2.0f);
+    EnDg_PlaySfxGrowl(this, 3.0f);
+    EnDg_PlaySfxBark(this, 5.0f);
 }
 
-void func_8098B464(EnDg* this, PlayState* play) {
-    Vec3f sp2C;
+/**
+ * Makes a splash effect and plays the splash SFX when the dog enters the water, then
```

Comment:
```suggestion
 * Makes a splash effect and plays the splash sfx when the dog enters the water, then
```
I'm not sure if it should be `Sfx` or `sfx`, but it's an abbreviation, not an acronym